src/templates/legal/risk_disclaimer.html.tt

```diff
@@ -0,0 +1,30 @@
+
+    <h2>[% l("Leverage") %]</h2>
+    <p>[% l("Leverage, in the context of margin trading, refers to the use of margin to increase/amplify the value of a trade. Leverage is expressed as a ratio. A 50:1 leverage, for example, means that you would be able to enter into a trade position that is of 50 times higher than your margin. When you invest in a leveraged product, your profit or loss is based on the leveraged position, i.e., the increased/amplified value of your trade determined based on the leverage ratio you set for your account.") %]</p>
+    <p>[% l("Leverage can be used to generate returns that are of multiple times higher than the amount set aside from your account balance to open and maintain a transaction (your margin). However, investments that have the potential to deliver greater investment returns carry higher levels of risk. This means that if the price movement is against your favour, you will suffer losses that are of multiple times higher than the amount set aside from your account balance to open and maintain a transaction (your margin).") %]</p>
+    <p>[% l("If you do not intend to utilise the leverage feature, please set the leverage ratio to 1:1.") %]</p>
+
+    <h2>[% l("Relationship between Margin and Leverage") %]</h2>
+    <p>[% l("The leverage ratio refers to the number of times your margin (the amount set aside from your account balance to open and maintain a transaction) can be \"leveraged\" to enter into a Transaction. The denominator of the leverage ratio is always 1. A 50:1 leverage is equivalent to a 2% margin requirement (1 divided by 50 = 2%). The leverage ratio that we/you establish will determine the amount of margin required. As leverage is inversely proportional to margin, the lower the leverage you set for your Account, the higher the margin required to be set aside from your account balance (your margin requirement).") %]</p>
+
+    <h2>[% l("Risk of Stop Order") %]</h2>
+    <p>[% l("The exchange fills orders on a first-in-first out basis, i.e. based on the sequence of the order received, and this is beyond our control. As a result, you may find it difficult or impossible to close a position at the intended price stipulated in your Stop Order during certain market conditions. The placement of Stop Order by you, though useful as a risk management tool, will not necessarily limit your losses to your intended amounts, in the event of highly volatile trading conditions, since market conditions may make it impossible to execute such orders and there is uncertainty about your possible exit price. In short, a Stop Order is not guaranteed because it cannot operate whatever the market circumstances are. Guaranteed Stop Orders are available at a premium price if you intend to avoid this risk.") %]</p>
```

Comment:
Remove the sentence "Guaranteed Stop Orders are available at a premium price if you intend to avoid this risk." altogether.